From 4cd696912d0cd9175eb7bec9cccc117e554207e2 Mon Sep 17 00:00:00 2001
From: Richard Levitte <levitte@openssl.org>
Date: Mon, 8 Apr 2024 15:14:40 +0200
Subject: [PATCH 065/228] doc/fingerprints.txt: Add the future OpenSSL release
 key

This will be used for future releases

Reviewed-by: Tomas Mraz <tomas@openssl.org>
Reviewed-by: Matt Caswell <matt@openssl.org>
(Merged from https://github.com/openssl/openssl/pull/24063)

(cherry picked from commit 4ffef97d3755a0425d5d72680daebfa07383b05c)
---
 doc/fingerprints.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/doc/fingerprints.txt b/doc/fingerprints.txt
index 9a26f7c667..9613cbac98 100644
--- a/doc/fingerprints.txt
+++ b/doc/fingerprints.txt
@@ -15,6 +15,9 @@ currently in use to sign OpenSSL distributions:
 OpenSSL OMC:
 EFC0 A467 D613 CB83 C7ED 6D30 D894 E2CE 8B3D 79F5
 
+OpenSSL:
+BA54 73A2 B058 7B07 FB27 CF2D 2160 94DF D0CB 81EF
+
 Richard Levitte:
 7953 AC1F BC3D C8B3 B292 393E D5E9 E43F 7DF9 EE8C
 
-- 
2.39.5 (Apple Git-154)

